CHANGELOG.md

```diff
@@ -0,0 +1,689 @@
+
+  Bolt now sends an analytics event when the built-in inventory plugins are used. ([BOLT-1410](https://tickets.puppet.com/browse/BOLT-1410))
+
+#### Bug fixes
+
+* **Bolt debug output showed task and script arguments as Ruby hashes, not JSON**
+
+  Bolt debug output now prints task and script arguments as JSON instead of Ruby hashes. ([BOLT-1456](https://tickets.puppet.com/browse/BOLT-1456))
+* **out::message didn't print when format=json**
+
+  The `out::message` standard plan function now prints messages as expected even when it is configured to use JSON. ([BOLT-1455](https://tickets.puppet.com/browse/BOLT-1455))
+
+## 1.26.0
+
+#### New features
+
+* **Options for PCP transport now configurable in bolt.yaml**
+
+  The job-poll-timeout and job-poll-interview options for the PCP transport are now configurable in bolt.yaml. ([BOLT-1425](https://tickets.puppet.com/browse/BOLT-1425))
+* **Task plugin improvements**
+
+  The task plugin now enables you to run a task to discover targets or look up configuration information in the version 2 inventory file. ([BOLT-1408](https://tickets.puppet.com/browse/BOLT-1408))
+* **Ability to see nodes in an inventory group**
+
+  You can now see what nodes a Bolt command acts on using the bolt inventory show subcommand. Pass a targeting option, such as -n node1,node2, -n groupname, -q query, --rerun, and other targeting options to specify which nodes to list. ([BOLT-1398](https://tickets.puppet.com/browse/BOLT-1398))
+* **Support for an apply step**
+
+  YAML plans now support applying Puppet resources with a resources step. ([BOLT-1222](https://tickets.puppet.com/browse/BOLT-1222))
+
+#### Bug fixes
+
+* **Modulepath now handles folder names in uppercase characters on Windows**
+
+  Bolt now prints a warning stating that it is case sensitive when the specified path is not found but another path is found with different capitalization. For example, if the actual path is C:\User\Administrator\modules but the user specifies C:\user\administrator\modules, a warning states that the specified path was not used and that the correct path is C:\User\Administrator\modules. ([BOLT-1318](https://tickets.puppet.com/browse/BOLT-1318))
+
+## 1.25.0
+
+#### Bug fixes
+
+* **out::message didn't work inside without_default_logging**
+
+  The out::message standard library plan function now works within a without_default_logging block. ([BOLT-1406](https://tickets.puppet.com/browse/BOLT-1406))
+* **Task action stub parameter method incorrectly merged options and arguments**
+
+  When a task action stub expectation fails, the expected parameters are now properly displayed. ([BOLT-1399](https://tickets.puppet.com/browse/BOLT-1399))
+
+#### Deprecations and removals
+
+* **lookups removed from target_lookups**
+
+  We have deprecated the target-lookups key in the experimental inventory file v2. To address this change, migrate any target-lookups entries to targets and move the plugin key in each entry to _plugin. 
```

Comment:
Looks like the `_` tricks the code diff preview to thinking this is _itallic_. Is it work trying to do some find-replace work to escape with \_itallic\_. 
The difference above is
```
_itallic_
\_itallic\_
```